### ISSUE TITLE
Using DecodeHeader method

### ISSUE THREAD

**Issue by stefanstranger** [jwt-dotnet/jwt#276]
I want to decode the header info of Json Web Token via the [DecodeHeader](https://github.com/jwt-dotnet/jwt#parsing-decoding-token-header) Method in PowerShell.

But I don't see this method after loading the jwt.dll.

```PowerShell
Import-Module -Name C:\Users\stefstr\Documents\GitHub\PSJwt\lib\jwt\net48\JWT.dll
Import-Module -Name C:\Users\stefstr\Documents\GitHub\PSJwt\lib\Newtonsoft\net45\Newtonsoft.Json.dll

$Serializer = [JWT.Serializers.JsonNetSerializer]::new()
$UrlEncoder = [JWT.JwtBase64UrlEncoder]::new()
$Decoder = [JWT.JwtDecoder]::new($Serializer, $UrlEncoder)

$Decoder| Get-Member -MemberType Method

$token = 'eyJmb28iOiJiYXIiLCJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJEZW1vIjoiRW5jb2RlIEFjY2VzcyBUb2tlbiIsImlhdCI6IjEzOTMyNjg4OTMiLCJMYXN0TmFtZSI6IlN0cmFuZ2VyIiwiZXhwIjoiMTM5MzI4Njg5MyIsIkZpcnN0TmFtZSI6IlN0ZWZhbiJ9.bu3cFwBzdhzflWW9AYTX1N0tn2RL2QJE_FLLu9OPsi8'

$Decoder.Decode($token)
```

How can I use the DecodeHeader method?

Regards,
Stefan Stranger

**Comment by stefanstranger** [comment]
I just solved the issue by using the latest release version. My initial binaries came from the nuget package jwt and was version 6.1.4.

The latest 7.1.0 version has the new DecodeHeader method.!

When will you update the nuget package?

/Stefan

**Comment by abatishchev** [comment]
Will publish as soon as sort of few outstanding issue, like #277. If that one is resolved by 7.1 - it's a good reason to publish it.

**Comment by abatishchev** [comment]
7.1.0 has been released. Please try and let me know if you see any issues.
